packages/ses/src/module-proxy.js

```diff
@@ -40,7 +47,7 @@ export const deferExports = () => {
           throw new TypeError(
             `Cannot get property ${q(
               name,
-            )} of module exports namespace, the module has not yet begun executing`,
+            )} of module exports namespace, the module has not yet finished executing`,
```

Comment:
Depending on our resolution, maybe please change this back.

Reply:
Thank you for pointing out that `import` gets called before executing the body of the module. That fixed my thinking about how this works. I’m not committed to preventing the hazard I described.